br/pkg/restore/internal/rawkv/rawkv_client.go

```diff
@@ -92,7 +93,10 @@ func (c *RawKVBatchClient) Put(ctx context.Context, key, value []byte, originTs
 			keys = append(keys, kv.key)
 			values = append(values, kv.value)
 		}
+		start := time.Now()
 		err := c.rawkvClient.BatchPut(ctx, keys, values, rawkv.SetColumnFamily(c.cf))
+		metrics.RawKVBatchPutBatchSize.WithLabelValues(c.cf).Observe(float64(len(c.kvs)))
```

Comment:
call `WithLabelValues` frequently may cause some performance drop. Please keep an eye on performance change after the PR. Or cache the result of `WithLabelValues` as a field of `c *RawKVBatchClient`